e2e_tests/tests/experiment/test_custom_searcher.py

```diff
@@ -582,3 +584,102 @@ def save_state(self, experiment_id: int, operations: List[searcher.Operation]) -
                 connectionpool.HTTPConnectionPool(host="dummyhost", port=8080), "http://dummyurl"
             )
             raise ex
+
+
+@pytest.mark.e2e_cpu
+def test_archived_proj_exp_list() -> None:
+    session = api_utils.determined_test_session(admin=True)
+    workspaces: List[bindings.v1Workspace] = []
+    count = 2
+
+    for _ in range(count):
+        body = bindings.v1PostWorkspaceRequest(name=f"workspace_{uuid.uuid4().hex[:8]}")
+        workspaces.append(bindings.post_PostWorkspace(session, body=body).workspace)
+
+    projects = []
+    experiments = []
+    for i in workspaces:
+        body1 = bindings.v1PostProjectRequest(name=f"p_{uuid.uuid4().hex[:8]}", workspaceId=i.id)
+        pid1 = bindings.post_PostProject(
+            session,
+            body=body1,
+            workspaceId=i.id,
+        ).project.id
+
+        body2 = bindings.v1PostProjectRequest(name=f"p_{uuid.uuid4().hex[:8]}", workspaceId=i.id)
+        pid2 = bindings.post_PostProject(
+            session,
+            body=body2,
+            workspaceId=i.id,
+        ).project.id
+
+        projects.append(pid1)
+        projects.append(pid2)
+
+        experiments.append(
+            exp.create_experiment(
+                conf.fixtures_path("no_op/single.yaml"),
+                conf.fixtures_path("no_op"),
+                ["--project_id", str(pid1)],
+            )
+        )
+        experiments.append(
+            exp.create_experiment(
+                conf.fixtures_path("no_op/single.yaml"),
+                conf.fixtures_path("no_op"),
+                ["--project_id", str(pid1)],
+            )
+        )
+        experiments.append(
+            exp.create_experiment(
+                conf.fixtures_path("no_op/single.yaml"),
+                conf.fixtures_path("no_op"),
+                ["--project_id", str(pid2)],
+            )
+        )
+        experiments.append(
+            exp.create_experiment(
+                conf.fixtures_path("no_op/single.yaml"),
+                conf.fixtures_path("no_op"),
+                ["--project_id", str(pid2)],
+            )
+        )
+
+    for x in experiments:
+        exp.wait_for_experiment_state(experiment_id=x, target_state=experimentv1State.COMPLETED)
```

Comment:
suggestion: I don't think we need to wait for the exps to complete. can we just kill them immediately? if that's the case, I think you can also pass the `--paused` argument to create the experiments in the paused state.

Reply:
If I killed without waiting, it was throwing an error while archiving saying that the experiment wasn't in terminal state. But I found a way to make it faster by killing it first and then waiting for target state CANCELED instead